### PR TITLE
chore: add icons

### PR DESCRIPTION
**Pull request by iamnamananand996**
Because

### need to add icons
- Speaker01
- VideoRecorder
- Anthropic
- Cohere
- Milvus
- Pinecode

This commit

### added icons
- Speaker01
- VideoRecorder
- Anthropic
- Cohere
- Milvus
- Pinecode

### CODE REVIEW

File: packages/design-system/src/new-ui/Icons/Dot.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+>((props, ref) => {
+  const { className, ...passThrough } = props;
+  return (
+    <IconBase
+      {...passThrough}
+      ref={ref}
+      viewBox="0 0 24 24"
+      className={className}
+    >
+      <circle cx="5" cy="5" r="4" fill="white" />
```
Comment (EiffelFly) [inline packages/design-system/src/new-ui/Icons/Dot.tsx:16]: @iamnamananand996 

Fill is not working in this icons set, all the icons are doing thing with path, please move this icons to ComplicatedIcons for a temp solution

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 Please follow up suggested changes

File: packages/design-system/src/new-ui/Logos/Anthropic.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      viewBox="0 0 60 60"
+      className={className}
+    >
+      <path
+        d="M34.9344 10L50.8819 50H59.6273L43.6798 10H34.9344Z"
+        fill="#181818"
+      />
+      <path
+        d="M16.4332 34.1714L21.8899 20.1143L27.3466 34.1714H16.4332ZM17.3178 10L1.3728 50H10.2883L13.5493 41.6H30.231L33.4915 50H42.407L26.4619 10H17.3178Z"
+        fill="#181818"
```
[the remark this answers is not in the document]
Comment (iamnamananand996) [reply packages/design-system/src/new-ui/Logos/Anthropic.tsx:22]: here, I think we should not remove the fill, the logo will on same color always.

[the remark this answers is not in the document]
Comment (EiffelFly) [reply packages/design-system/src/new-ui/Logos/Anthropic.tsx:22]: You are right

File: packages/design-system/src/new-ui/Icons/Dot.tsx (hunk 1 above)
Comment (EiffelFly) [reply packages/design-system/src/new-ui/Icons/Dot.tsx:16]: @iamnamananand996 Please delete this dot icon and we are good to go

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 Let delete the dot icon

**Review by EiffelFly — APPROVED** [review]
LGTM

**Pull request merged**